sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedResponse.java

```diff
@@ -21,6 +24,9 @@
      * @return The items in the page.
      */
     default List<T> getValue() {
```

Comment:
Maybe we can add @Deprecated annotation to this method too? This is redundant with the `getItems()` method  in `Page` this interface extends.

Reply:
Please see the comment: https://github.com/Azure/azure-sdk-for-java/pull/6465#discussion_r363119218